client/src/App.css

```diff
@@ -35,4 +35,4 @@
   to {
     transform: rotate(360deg);
   }
-}
+} */
```

Comment:
so what becomes of this file?
find anything online that says we should keep or remove?
if we're keeping for now, let's leave a `// TODO find out if we still need this or can remove`

Reply:
Ok. I think we can delete it since it's not used, but I wanted to be cautious

Reply:
I think we can remove it, but I didn't want to be hasty. Adding a note is good.